packages/api/src/sequelize/migrations/2023-12-12_00_create-patient-discovery.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import * as shared from "../migrations-shared";
+
+const tableName = "patient_discovery_result";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await shared.createTable(
+      queryInterface,
+      tableName,
+      {
+        id: {
+          type: DataTypes.STRING,
+          primaryKey: true,
+        },
+        requestId: {
+          type: DataTypes.UUID,
+          field: "request_id",
+        },
+        patientId: {
+          type: DataTypes.UUID,
+          field: "patient_id",
+        },
+        status: {
+          type: DataTypes.STRING,
+          field: "status",
+        },
+        data: {
+          type: DataTypes.JSONB,
+        },
+      },
+      { transaction, addVersion: true }
```

Comment:
Likely don't need version on this table, right?

Actually, might consider whether we even need `updatedAt` - if not, either don't use `shared.createTable()` or update it to allow `updatedAt` to be parameterized like `version`.

Reply:
Good point will update